modules/tools/mapviewers/README.md

```diff
@@ -32,3 +32,29 @@ The default utm zone is set to ***10*** in this tool. if the HDMap is located in
 python modules/tools/mapviewer/gmap_viwer.py map_path_and_file utm_zone_id
 ```
 
+### hdmapviewer.py
+Activate enviroments: 
+
+```
+conda create --name py27bokeh python=2.7.15 numpy scipy bokeh protobuf
```

Comment:
Do you want to merge the conda environment with the one under https://github.com/ApolloAuto/apollo/blob/master/docker/build/installers/py27_conda.yml for easier management? Or at least create a  .yml file?